gensim/models/word2vec.py

```diff
@@ -606,7 +606,7 @@ def load_word2vec_format(cls, fname, fvocab=None, binary=False, norm_only=True):
                     result.syn0[line_no] = fromstring(fin.read(binary_len), dtype=REAL)
             else:
                 for line_no, line in enumerate(fin):
-                    parts = utils.to_unicode(line).split()
+                    parts = utils.to_unicode(line.rstrip()).split(" ")
```

Comment:
Are you still talking about `rstrip`? I'm getting confused.

`rstrip` will remove ANY whitespace from the end of a string. We DON'T want that. We want to remove only `\n` (or whatever character C word2vec puts at the end of each line). Want want to keep any other (weird) whitespace intact.

Btw simulating split ala `(ch == ' ') || (ch == '\t') || (ch == '\n')` is just `[part for part in re.split((r"[ \t\n]", x) if part]` in Python.